app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,9 @@
+class GuestRoomsController < ApplicationController
+
+  def add_room
+    guest_id = params[:guest_id]
+    room_id = params[:room_id]
+    @guest_room = GuestRoom.create(guest_id: guest_id, room_id: room_id)
+    redirect_to "/guests/#{guest_id}"
+  end
+end
```

Comment:
Everything about this is good except that this should be in a `create` action, not a custom `add_room` action.